.gemini-chrome.yml

```diff
@@ -0,0 +1,16 @@
+rootUrl: http://localhost:8080/components/vaadin-checkbox/test/visual/
+gridUrl: http://localhost:4444/wd/hub
+screenshotsDir: ./test/visual/screens
+
+system:
+  plugins:
+    polyserve:
+      port: 8080
+    sauce: true
+
+browsers:
+  chrome:
```

Comment:
For some weird reason Chrome needs to be run separately from the rest of the browsers. When I had `Chrome` in `.gemini.yml` the screenshots shows only the content of light dom (as in elements were not imported at all)